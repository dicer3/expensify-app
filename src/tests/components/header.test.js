import ReactshallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Header from '../../components/header';

test('should render header correctly',()=>
{
    const renderer=new ReactshallowRenderer();
   renderer.render(<Header />)
   expect(renderer.getRenderOutput()).toMatchSnapshot();
    //console.log(renderer.getRenderOutput())

})