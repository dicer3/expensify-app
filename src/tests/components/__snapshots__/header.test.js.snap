// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    expensify
  </h1>
  <NavLink
    activeClassName="is_active"
    aria-current="page"
    exact={true}
    to="/"
  >
    dashboard
  </NavLink>
  <NavLink
    activeClassName="is_active"
    aria-current="page"
    to="/create"
  >
    create page
  </NavLink>
</header>
`;
